samples/CMakeLists.txt

```diff
@@ -107,21 +108,6 @@ if (CONFIG_BT_RPMSG_NRF53)
     message(FATAL_ERROR
       "SoC not supported for including hci_rpmsg as child image")
   endif()
-elseif (CONFIG_NRF_802154_SER_HOST)
```

Comment:
Why is 802154_rpmsg being removed?

Reply:
It's not working after last changes. Is it still supported?

Reply:
Yes, it should still be working.